libass/ass_render.c

```diff
@@ -34,7 +34,12 @@
 #define MAX_BITMAPS_INITIAL 16
 #define MAX_SUB_BITMAPS_INITIAL 64
 #define SUBPIXEL_MASK 63
-#define SUBPIXEL_ACCURACY 7
+#define STROKER_ACCURACY 16     // stroker error in integer units
```

Comment:
Nit: I think it'd be more correct to refer call these `_PRECISION`.

Reply:
Done